gamemodes/terrortown/gamemode/shared/lang/english.lua

```diff
@@ -1221,6 +1221,8 @@ L.ttt2_bindings_new = "New bound key for {name}: {key}"
 
 L.f1_bind_weaponswitch = "Switch Weapon"
 L.f1_bind_sprint = "Sprint"
+L.f1_bind_voice = "Voice Chat"
```

Comment:
maybe name this `General Voice Chat`? Just an idea though

Reply:
i second naming it more precisely, although i personally would prefer `Global Voice Chat`

Reply:
It's not necessarily global though due to locational voice chat

Reply:
That might be true - I actually don't know as I have not used the ingamevoice in a long time - but i feel like that does not really matter as having an unrestricted chat (everyone has access to) be called "Global" is a reoccurring thing in video games.
On the other hand I can definitely see where you are coming from and it might be alright as it is

Reply:
Done